autometrics/tests/error_locus.rs

```diff
@@ -0,0 +1,7 @@
+//! Tests for the ResultLabels macro
```

Comment:
why is this a separate trybuild test cases collection instead of just adding it to the existing ones?

Reply:
I should change the comment btw

I made this a separate case mostly so that it appears differently in cargo test output, so we know what’s tested. But I can merge the trybuild tests and just rename the file from `result_labels` to `compilation_tests` and that’d be fine too, I’ll do this I think